### PR TITLE
NOTICE file added

### PR DESCRIPTION
**Pull request by marcin-krolik**
NOTICE file with license info

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,5 @@
+snap-plugin-collector-docker
```
Comment (katarzyna-z) [inline NOTICE]: Here you should replace "snap-plugin-collector-docker" with "Intel Snap"

File: LICENSE [hunk 2]
```diff
@@ -175,3 +175,8 @@
       of your accepting any such warranty or additional liability.
 
    END OF TERMS AND CONDITIONS
+
+
+This product contains code from Rackspace GopherCloud, which is available
+under an Apache License 2.0. For details, see openstack/v2/snapshots/requests.go, openstack/v2/snapshots/results.go,
+openstack/v2/volumes/requests.go, openstack/v2/volumes/results.go
```
Comment (katarzyna-z) [inline LICENSE]: In mentioned files (openstack/v2/snapshots/requests.go, openstack/v2/snapshots/results.go, openstack/v2/volumes/requests.go, openstack/v2/volumes/results.go) if there are original to Intel but with snippets from 3rd party components - files should have two Copyrights headers: Intel + 3rd component, see [this](https://github.com/intelsdi-x/snap-plugin-publisher-heapster/blob/master/exchange/exchange.go) example.

**Comment by mkleina** [comment]
@katarzyna-z I see that you added copyright headers to source files, can it be merged now?

**Comment by katarzyna-z** [comment]
There should be added information about parts of code which come from 3rd party components so this pull request should not be merge.

**Comment by padamowi** [comment]
LGTM

**Comment by katarzyna-z** [comment]
LGTM

**Pull request merged**